tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "skipLibCheck": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
-    "target": "ES2022"
+    "target": "ES2022",
+    "module": "ES2022"
```

Comment:
`module` must be set for dynamic imports to work, I just follow the `target` for the value

Reply:
`ES2023` is not yet a valid value for `module` so I had to use `ES2022`.